server.js

```diff
@@ -1,33 +1,71 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
 
 // If you're using one of our datasets, uncomment the appropriate import below
 // to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
+//
+import goldenGlobesData from './data/golden-globes.json';
+
+// import avocadoSalesData from './data/avocado-sales.json';
 // import booksData from './data/books.json'
 // import netflixData from './data/netflix-titles.json'
 // import topMusicData from './data/top-music.json'
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+	res.send('Hello world');
+});
+
+app.get('/nominations', (req, res) => {
+	res.json(goldenGlobesData);
+});
+
+app.get('/year/:year', (req, res) => {
+	const year = req.params.year;
+	const showWin = req.query.win;
+	let nominationsFromYear = goldenGlobesData.filter(
+		(item) => item.year_award === +year
+	);
+
+	if (showWin) {
+		nominationsFromYear = nominationsFromYear.filter((item) => item.win);
+	}
+	res.json(nominationsFromYear);
+});
+
+//single item
+
+app.get('/year/:year/category/best_drama/?win=true', (req, res) => {
+	const year = req.params.year;
+	const showWin = req.query.win;
+
+	let bestDrama = goldenGlobesData.filter(
+		(item) => item.category === 'Best Motion Picture - Drama'
```

Comment:
Would be nice if this was also a query param under the `nominations` endpoint 🤩 
`nominations?year=2010&win=true?category=drama`

Reply:
Oh, this endpoint doesn't seem to be working 🤔 `?win=true` shouldn't be in the naming of the endpoint, but rather just a query param - right?

Reply:
Changed the endpoint and it's working now

Reply:
Looks like you removed it, or did you not commit your latest changes maybe. The requirements say to have one endpoint that only shows a single result

Reply:
Fixed it now, changed dataset to have an ID to search for to show a single item